### ISSUE TITLE
Tablesaw seriliaze null String values

### ISSUE THREAD

**Issue by yelizgungor** [jtablesaw/tablesaw#1221]
While serializing tablesaw to CSV to String, I use CsvWriter. Default [nullValue](https://github.com/jtablesaw/tablesaw/blob/master/core/src/main/java/tech/tablesaw/io/csv/CsvWriter.java#L33) is used as the empty string. I need to set this nullValue parameter with CsvWriteOptions to set a specific string Ex: "__null_rep" or "__na_rep" as python pandas dataframe to_csv method serialize with [na_rep](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html) parameter.

```
Table table = createNewTable();
ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
CsvWriteOptions options = CsvWriteOptions.builder(byteArrayOutputStream)
            .header(true)
            .build();
 new CsvWriter().write(table, options)
 return byteArrayOutputStream.toByteArray();
```
@question

**Comment by lwhite1** [comment]
you can change the value before writing using one of the set methods
defined on the column

**Comment by yelizgungor** [comment]
Yup, I will do that as you suggest in [this issue](https://github.com/jtablesaw/tablesaw/issues/564). But it would be good to have a setting for that. 
Thanks for response, @lwhite1

**Comment by lwhite1** [comment]
actually there is a setting, you can elect when writing to use a
PrintFormatter (I forget the exact class name). You should be able to set a
value for null there.

**Comment by yelizgungor** [comment]
I updated the StringColumns formatter. But it seems [CsvWriter](https://github.com/jtablesaw/tablesaw/blob/e11ef19d035c409367a2a05d6a8301188edf9387/core/src/main/java/tech/tablesaw/io/csv/CsvWriter.java#L83) calls [getUnformattedString](https://github.com/jtablesaw/tablesaw/blob/e11ef19d035c409367a2a05d6a8301188edf9387/core/src/main/java/tech/tablesaw/api/StringColumn.java#L614) instead [getString](https://github.com/jtablesaw/tablesaw/blob/e11ef19d035c409367a2a05d6a8301188edf9387/core/src/main/java/tech/tablesaw/api/StringColumn.java#L608-L610)

**Comment by yelizgungor** [comment]
But AFAIS there is another main problem: There is no differentiation for StringColumn between an empty string and a null value to define if it  [isMissing](https://github.com/jtablesaw/tablesaw/blob/e11ef19d035c409367a2a05d6a8301188edf9387/core/src/main/java/tech/tablesaw/columns/AbstractColumnParser.java#L36-L41)

**Comment by lwhite1** [comment]
In tablesaw an empty string is a null value is a missing value.
